using-pagination/src/pages/index.js

```diff
@@ -1,13 +1,40 @@
+import Link from 'next/link';
 import { gql } from 'graphql-request';
 
 import { graphcms } from '../../lib/_graphcms';
 
 const limit = 1;
 
+const singleProductStyle = {
+  display: 'flex',
+  flexDirection: 'column',
+  width: '300px',
+  margin: '10px',
+};
+
+function SingleProduct({product, index}) {
+  return (
+    <div style={singleProductStyle}>
+      <img src={`https://via.placeholder.com/300x200?text=${product.name}`} width={300} height={200} alt={product.name} title={product.name} /> 
```

Comment:
Why are we using placeholder? The project has images with each product.

Reply:
The original code didn't have images in it, so I didn't know. 😅 Could you please invite me to the project?